sdk/tests/deqp/functional/gles3/es3fTextureSpecificationTests.js

```diff
@@ -1677,7 +1677,7 @@ goog.scope(function() {
         );
 
         // Fill data with grid.
-        data = new ArrayBuffer(rowPitch * height);
+        data = new ArrayBuffer(rowPitch * height + this.m_skipPixels * pixelSize);
```

Comment:
This is a bug in the original c++ code. Without fixing this, the skippixels test still pass, but if we test with a different skippixels value (say a large number like 100), then it fails with this fix.